app/admin/category/index/template.hbs

```diff
@@ -0,0 +1,12 @@
+<p class="title">Categories</p>
+<div class="options">
+  <div class="options__row options__row--end">
+    {{#link-to "admin.category.new" class="category-btn" }}Add New{{/link-to}}
```

Comment:
You will need to pass in the `model.id` here so that the `:id` part of the url can be filled in for this link.